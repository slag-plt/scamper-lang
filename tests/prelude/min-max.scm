(min 4 7 2)
(min 0.01 0.2 0.5)
(min 100 10 1)
(min 2.0 2)
(min 3.0 3.1)
(max 4 7 2)
(max 0.01 0.2 0.5)
(max 100 10 1)
(max 2.0 2)
(max 3.0 3.1)