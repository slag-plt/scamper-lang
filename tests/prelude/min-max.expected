2
0.01
1
2
3
7
0.5
100
2
3.1
